e2e/tests-dfx/sns.bash

```diff
@@ -47,3 +47,31 @@ SNS_CONFIG_FILE_NAME="sns.yml"
     assert_command_fail dfx sns config validate
     assert_match token.name
 }
+
+@test "sns deploy exists" {
+    dfx sns deploy --help
+}
+
+@test "sns deploy fails without config file" {
+    dfx_new
+    rm -f sns.yml # Is not expected to be present anyway
+    assert_command_fail dfx sns deploy
+    assert_match 
+}
+
+@test "sns deploy succeeds" {
+    dfx_new
+    install_shared_asset subnet_type/shared_network_settings/system
+    dfx start --clean --background --host 127.0.0.1:8080
+    sleep 1
+    dfx nns install
+    # There are no entries for "local" upstream yet, so we need a network mapping.
+    dfx nns import --network-mapping local=mainnet
+    # This canister ID is not included upstream .. yet.
+    jq '.canisters["nns-sns-wasm"].remote.id.local="qjdve-lqaaa-aaaaa-aaaeq-cai"' dfx.json | sponge dfx.json
+    ls candid
+    cat dfx.json
+    cp "${BATS_TEST_DIRNAME}/../assets/sns/valid_sns_init_config.yaml" "$SNS_CONFIG_FILE_NAME"
+    cp "${BATS_TEST_DIRNAME}/../assets/sns/logo.svg" .
```

Comment:
The `install_asset` function can help here.  I'd recommend something like
```
install_asset sns/valid
install_asset sns/logo
```

and move these files to `e2e/assets/sns/valid/sns.yml` and `e2e/assets/sns/logo/logo.svg`

Reply:
Can I put these assets in a single directory, as they will normally be needed together?  Then the call would be `install_asset sns` ?

Reply:
Yes!